controller/Controller.py

```diff
@@ -0,0 +1,207 @@
+# Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script implementing a remote controller for the DDS Recorder."""
+
+from enum import Enum
+
+from PyQt6.QtCore import QObject, pyqtSignal
+
+from ddstypes.ControllerCommand.ControllerCommand import (
+    ControllerCommand, ControllerCommandPubSubType)
+from ddstypes.Status.Status import Status, StatusPubSubType
+
+import fastdds
+
+from utils.Logger import Logger
+
+
+class DdsRecorderStatus(Enum):
+    RECORDING = 1
+    PAUSED = 2
+    STOPPED = 3
+    UNKNOWN = 4
```

Comment:
Other status unavailable or not_existant?
Depends if this status is for internal purpose or for the user info